src/System.Management.Automation/engine/remoting/commands/EnterPSHostProcessCommand.cs

```diff
@@ -91,6 +92,16 @@ public PSHostProcessInfo HostProcessInfo
             set;
         }
 
+        /// <summary>
+        /// Gets or sets the Named Pipe name to connect to.
```

Comment:
Should we mention "debug" here?

Reply:
fixed this with rename to CustomPipeName